src/Preflight/Preflight.php

```diff
@@ -284,8 +284,8 @@ public function preflight($argv)
         // report on it from Drush status or wherever else it may be needed.
         $configFilePaths = $this->configLocator->configFilePaths();
         $config->set('runtime.config.paths', $configFilePaths);
-        $this->logger()->log(dt('Config paths: ' . implode(',', $configFilePaths)));
-        $this->logger()->log(dt('Alias paths: ' . implode(',', $paths)));
+        $this->logger()->log(dt('Config paths: ' . implode(',', (array)$configFilePaths)));
```

Comment:
This cast has a slight smell. Why would configFilePaths() sometimes return an array and sometimes not?

Reply:
There was a bug. I can take that out.